client/my-sites/site-monitoring/metrics-tab.tsx

```diff
@@ -279,6 +279,7 @@ export const MetricsTab = () => {
 						label: __( 'Average response time (ms)' ),
 						stroke: '#008763',
 						scale: 'average-response-time',
+						unit: 'ms',
```

Comment:
I think units might need to be translated in some languages

Reply:
I tried looking for examples in our codebase but couldn't find anything. Did you see any similar case?

If we decide to translate it, we would likely need to translate the full string e.g. `translate( '%(value) ms' )`.

Reply:
It looks like it's 'ms' across most languages, except the non-Latin character ones: https://indifferentlanguages.in/word/english/743f2b8/millisecond

We'll run into a spacing issue with non-Latin alphabets. I think they'll recognize 'ms' as a unit, so let's keep the string as it is for now.

Reply:
Thanks for researching it more @danielbachhuber, sounds good to leave it as is now.